controller/src/beerocks/bml/internal/bml_internal.cpp

```diff
@@ -83,6 +83,9 @@ bml_internal::~bml_internal() {}
 
 int bml_internal::send_bml_cmdu(int &result, uint8_t action_op)
 {
+    // initialize value
+    result = false;
```

Comment:
No, it is not a big deal change.
Just initializing the result integer to default value of "false", so if the function fails, it will return "false". If the function success it takes the value from the response.